src/encoded/static/components/cart/cart.js

```diff
@@ -115,40 +116,39 @@ const datasetTabTitles = {
 /**
  * Display browser tracks for the selected page of files.
  */
-const CartBrowser = ({ files, assembly, pageNumber, loading }) => {
-    if (!loading) {
-        // Extract the current page of file objects.
-        const pageStartingIndex = pageNumber * constants.PAGE_TRACK_COUNT;
-        const pageFiles = files.slice(pageStartingIndex, pageStartingIndex + constants.PAGE_TRACK_COUNT).map((file) => ({ ...file }));
-
-        // Shorten long annotation_type values for the Valis track label; fine to mutate `pageFiles` as
-        // it holds a copy of a segment of `files`.
-        pageFiles.forEach((file) => {
-            if (file.annotation_type) {
-                const mappedAnnotationType = annotationTypeMap[file.annotation_type];
-                if (mappedAnnotationType) {
-                    file.annotation_type = mappedAnnotationType;
-                }
-            }
-        });
-
-        const sortParam = ['Assay term name', 'Biosample term name', 'Output type'];
-        return <GenomeBrowser files={pageFiles} label="cart" assembly={assembly} expanded sortParam={sortParam} />;
+const CartBrowser = ({ files, assemblies, pageNumber, loading }) => {
+    if (loading) {
+        return <div className="cart__empty-message">Page currently loading&hellip;</div>;
     }
 
-    // Message for page loading.
-    if (loading) {
-        return <div className="nav result-table cart__empty-message">Page currently loading&hellip;</div>;
+    if (assemblies.length !== 1) {
+        return <div className="cart__empty-message">Select single assembly to view genome browser</div>;
```

Comment:
Probably, “Select **a** single assembly to view **the** genome browser.” But I don’t think this needs to be a complete grammatical sentence since it’s just a warning.